frontend/src/Editor/CodeBuilder/CodeHinter.jsx

```diff
@@ -269,9 +269,8 @@ export function CodeHinter({
   const [forceCodeBox, setForceCodeBox] = useState(fxActive);
   const codeShow = (type ?? 'code') === 'code' || forceCodeBox;
   cyLabel = paramLabel ? paramLabel.toLowerCase().trim().replace(/\s+/g, '-') : cyLabel;
-
   return (
-    <div ref={wrapperRef}>
+    <div ref={wrapperRef} className={codeShow ? 'component-action-select-codeShow' : ''}>
```

Comment:
This class is added in all of the places like `runJS` and places where codeHinter is being used. Can we try to avoid it?